### ISSUE TITLE
E800 shouldn't be triggered for `# noqa` lines

### ISSUE THREAD

**Issue by webknjaz** [wemake-services/wemake-python-styleguide#1457]
# Bug report

`# noqa` comments are wrongly recognized as commented out code.

## What's wrong

This
```python
# noqa: WPS437
mod._very_long_obj_name = _patched_very_long_obj_name
```
causes `E800: Found commented out code` and it's wrong. Sometimes
noqa comments don't fit in the line-limit violations and it's okay
to put them on the previous line.

## How is that should be

No error.

## System information

N/A

**Comment by sobolevn** [comment]
I am testing this out, but `noqa` does not seem to work this way:

```
# noqa: F401
from returns._generated.converters.flatten import _flatten as flatten
```

Output:

```
(.venv) ~/Documents/github/returns  master ✗                                    130 ⚠️
» flake8 .

./returns/converters.py

  4:2      E800: Found commented out code
  # noqa: F401
   ^

  5:1      F401  'returns._generated.converters.flatten._flatten as flatten' imported but unused
  from returns._generated.converters.flatten import _flatten as flatten
  ^

Full list of violations and explanations:
https://wemake-python-stylegui.de/en/0.14.0/pages/usage/violations/

```

**Comment by webknjaz** [comment]
@sobolevn I assume that it's up to a specific plugin (or its specific violation handler) how they parse `noqa`. Or maybe it's inconsistent. But I know that sometimes it works and sometimes it behaves weirdly.

**Comment by webknjaz** [comment]
I've done some googling and haven't found any mentions of using noqa on the previous line. So I might've confused it with pylint. FWIW _I think_ I saw this rule barking at pylint comments too.

**Comment by gram-eiq** [comment]
1. Flake8 doesn't support noqa on a separate line
1. Pylint does support it. Eradicate already has an exception for pylint: https://github.com/myint/eradicate/commit/d0098861348b0592222d59d8732eac848db14a3a

**Comment by orsinium** [comment]
1. Flake8 doesn't support noqa on a separate line
1. Pylint does support noqa on a separate line. Eradicate already has an exception for pylint: https://github.com/myint/eradicate/commit/d0098861348b0592222d59d8732eac848db14a3a

**Comment by sobolevn** [comment]
Done! https://github.com/myint/eradicate/blob/a7c2fcbaaa24a1cc0773babd88340a7a221a5971/eradicate.py#L57

New version will be released soon.
